ntp-proto/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "ntp-proto"
 version = "0.1.0"
 edition = "2021"
+license = "Apache-2.0/MIT"
```

Comment:
I think in order to distribute the licenses within the crates once published, you usually want to put a symlink to the license files in the repo root in the crate roots.